x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/api_key/10_grant.yml

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
I wasn't able to find any documentation around test file name prefixes: when do we use `10_` vs `11_` vs `20_` etc?

Reply:
There is likely a convention. But it was not clear to me either. My preferences/observations were:
1. Not reusing the same number
2. Choose the next number that is 10 apart, e.g. 10, 20, 30. So it leaves some space in case we need add 11, 21.
3. Numbering rougly corresponds when feature was added, e.g. 10 for the earliest.

If we use above "rules", I'd say `12` seems to fit.

Reply:
Are y'all too young to have programmed in BASIC?
https://en.wikipedia.org/wiki/Line_number#BASIC

Reply:
Lol does visual basic count?